packages/components/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": ["@babel/preset-typescript"],
+  "plugins": [
+    "@babel/plugin-transform-modules-commonjs",
```

Comment:
Is JSX still being transformed? I would expect to need to use either an explicit transform for that, or babel/preset-env. But let me know if is still somehow being transformed 🤔 

We may want to use babel/preset-env anyway, to ensure that we're being explicit about what environments we support (and transforming all syntaxes that we need to for those environments).

Reply:
Yeah, I was pairing with Annie, but was unable to see the git ignored files so I couldn't confirm. Since I'm taking over while Annie is out sick, I'll look into it.